app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +15,32 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-taxon {
```

Comment:
As a stylistic point, this is quite nested. I personally don't mind too much (especially since components shouldn't be concerned with CSS overriding), but I know some people can have strong opinions on this.